### ISSUE TITLE
[a11y]: Toggle component  fails a colour contrast check at disabled state

### ISSUE THREAD

**Issue by lw980219** [carbon-design-system/carbon#16851]
### Package

@carbon/react

### Browser

Firefox

### Operating System

MacOS

### Package version

v11

### React version

v17

### Automated testing tool and ruleset

IBM equal accessibility checker

### Description

https://react.carbondesignsystem.com/?path=/story/components-toggle–playground&args=disabled:!true
Toggle component cannot pass color contrast check when disabled=true

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-toggle–playground&args=disabled:!true

### Steps to reproduce

https://react.carbondesignsystem.com/?path=/story/components-toggle–playground&args=disabled:!true

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
This issue was linked to the Playground example. I tested the [component](https://react.carbondesignsystem.com/?path=/story/components-toggle--default&args=disabled:!true), and it's working correctly. If the problem happens again, we can either reopen this issue or create a new one.
